include/openthread/channel_manager.h

```diff
@@ -48,7 +48,9 @@ extern "C" {
  *   This module includes functions for Channel Manager.
  *
  *   The functions in this module are available when Channel Manager feature
- *   (`OPENTHREAD_CONFIG_CHANNEL_MANAGER_ENABLE`) is enabled. Channel Manager is available only on an FTD build.
+ *   (`OPENTHREAD_CONFIG_CHANNEL_MANAGER_ENABLE`) is enabled. Channel Manager is available on FTD build
+ *   and on MTD build with Csl active. On a FTD build it manages the network-wide Pan channel, and
+ *   on a MTD build the local Csl channel.
```

Comment:
Thanks @martinzi  for submitting this.

I agree with @Irving-cl suggestion belwo. I think may be better to add this as a new feature, say `OPENTHREAD_CONFIG_CHANNEL_MANAGER_CSL_CHANNEL_SELECTION_ENABLE`, which is limited to `MTD` builds and can then be enabled/disabled on its own.

Thoughts?

Reply:
I agree, it should be configurable. I added a new compiler switch for this. It also allows to fine tune behavior of FTD attached via csl.